project/Build.scala

```diff
@@ -123,21 +136,28 @@ object JobServerBuild extends Build {
         copy(artifact, artifactTargetPath)
         copy(baseDirectory(_ / "bin" / "server_start.sh").value, file("app/server_start.sh"))
         copy(baseDirectory(_ / "bin" / "server_stop.sh").value, file("app/server_stop.sh"))
+        copy(baseDirectory(_ / "bin" / "manager_start.sh").value, file("app/manager_start.sh"))
```

Comment:
Thanks.  I somehow thought we had added this fix before, but must have somehow slipped.... :-p